packages/injected/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/injected-wallets",
-  "version": "2.2.2-alpha.2",
+  "version": "2.2.2-alpha.3",
```

Not sure why release was merged into master but not into develop yet
```suggestion
  "version": "2.2.3-alpha.1",
```